odata-json-format/16 Bound Function.md

```diff
@@ -337,6 +337,11 @@ how to interpret the omitted parameter. Note: a nullable non-binding
 parameter is equivalent to being annotated as optional with a default
 value of `null`.
 
+Non-binding collection-valued parameters MAY be omitted from the request body regardless
+of nullability or optionality. The interpretation of such an omitted parameter is up to the service,
```

Comment:
"May be omitted" and "reporting an error" seem in conflict.  Can we remove the first part, and just say something like "If collection-valued parameters are omitted from the request, the interpretation is up to the service,..."